meta-luneui/classes/webos_filesystem_paths.bbclass

```diff
@@ -10,17 +10,20 @@ webos_defaultconfdir = "${sysconfdir}/default"
 webos_execstatedir = "${localstatedir}/lib"
 webos_fontsdir = "${datadir}/fonts"
 webos_homedir = "${base_prefix}/home"
-webos_firmwaredir = "${base_libdir}/firmware"
+webos_firmwaredir = "${nonarch_base_libdir}/firmware"
 webos_logdir = "${localstatedir}/log"
 webos_mediadir = "${base_prefix}/media"
 webos_mntdir = "${base_prefix}/mnt"
 # Discourage use of ${libdir}/pkgconfig by not providing a variable for it.
 webos_pkgconfigdir = "${datadir}/pkgconfig"
 webos_preservedtmpdir = "${localstatedir}/tmp"
 # Having a Qt plugins directory is standard, but the value used by webOS OSE isn't.
-# FIXME check if we can use new value
-# New value - webos_qtpluginsdir = "${libdir}/qt5/plugins"
-webos_qtpluginsdir = "${prefix}/plugins"
+# OE_QMAKE_PATH_QT_ARCHDATA might be undefined in components which don't inherit qmake5_paths
+# and cmake-modules-webos-native.bb:do_configure expects absolute path here
+# | CMake Error at webOS/webOS.cmake:316 (message):
+# |   ENV{webos_qtpluginsdir} is not an absolute path:
+# |   '${OE_QMAKE_PATH_QT_ARCHDATA}'
+webos_qtpluginsdir = "${libdir}/plugins"
```

Comment:
@Tofee @shr-project Not 100% sure about this change. It was taken like this from OSE. Will this be OK?

Reply:
If it points to the exact same path, I guess we're fine; maybe worth comparing with bb -E

Reply:
@tofee Yes seems so. Checked and old image and new image. End up in /usr/lib/plugins for both.